docs/docs/1.2.3/quickstart/universal.md

```diff
@@ -168,17 +221,7 @@ EOF
 
 This will enable the `prometheus` metrics backend on the `default` [Mesh](/docs/1.2.3/policies/mesh/) and automatically collect metrics for all of our traffic.
```

Comment:
Maybe we need to tell people they should run a prometheus when doing this?

Reply:
How much detail do you suggest? (I really cannot tell what the level of this doc should be ... any and all opinions welcome.)

Reply:
IMO this is a doc about Kuma so just mentionning they need prometheus is probably enough to start

Reply:
lol so I went to edit the prerequisites, and this is already in the doc (edited in latest commit to link to Prometheus docs):
```
To explore traffic metrics with the demo app, you also need to [set up Prometheus]((https://prometheus.io/docs/prometheus/latest/getting_started/)). See the [traffic metrics policy documentation](../../policies/traffic-metrics).
```